### ISSUE TITLE
Performing Purge operation along with delete operation on organisation of GUI 

### ISSUE THREAD

**Issue by iamarnavgarg** [ckan/ckan#4486]
### CKAN Version if known (or site URL)
2.7.2

In Reference with the Issue #4267 
As we know that deleting an organisation only changes the state of organisation from active to deleted state and is still accessible through the URL `http://<ckan-instance>/organisation/<name-of-org>` .

I would like to propose that can we add a new button on GUI for [organisation_purge](http://docs.ckan.org/en/latest/api/#ckan.logic.action.delete.organization_purge) API keeping delete also there OR else we can perform the purge operation by clicking on delete button. 

Please suggest for the issue as we cannot perform the organisation purge without firing the action api mentioned here.

Organization purge - curl http://url_of_ckan/api/action/organization_purge -H "X-CKAN-API-Key: x-x-x-x" -d '{"id": "name_of_org"}'

**Comment by amercader** [comment]
You are right in that there is no UI for purging organizations or datasets. I think that adding a "Purge" button next to the "Delete" one in the form would cause confusion to the majority of users that don't know the difference between them. Perhaps it makes more sense to add the ability to purge organizations and groups to the existing "Trash" tab in the admin interface (`/ckan-admin`).

**Comment by amercader** [comment]
Also please upgrade to 2.7.4 (latest patch release)

**Comment by mutantsan** [comment]
Hi, i have created a PR #5072 with implementation of organizations and groups purge
